src/video/SDL_egl.c

```diff
@@ -427,7 +429,7 @@ static bool SDL_EGL_LoadLibraryInternal(SDL_VideoDevice *_this, const char *egl_
 
     _this->egl_data->egl_dll_handle = egl_dll_handle;
 #ifdef SDL_VIDEO_DRIVER_VITA
-    _this->egl_data->opengl_dll_handle = opengl_dll_handle;
+    _this->egl_data->opengl_dll_handle = NULL;
```

Comment:
This seems wrong. opengl_dll_handle can be assigned above.

Reply:
`opengl_dll_handle` is only assigned in the block that's guarded by `#if !defined(SDL_VIDEO_STATIC_ANGLE) && !defined(SDL_VIDEO_DRIVER_VITA)`. If `SDL_VIDEO_DRIVER_VITA` is defined then it will remain at NULL.

Reply:
That data is zero initialized. It's probably clearer just to remove that `#ifdef` block.